docs/request.md

```diff
@@ -25,12 +25,12 @@ Makes XHR (aka AJAX) requests, and returns a [promise](promise.md)
 
 ```javascript
 m.request({
-	method: "PUT",
-	url: "/api/v1/users/:id",
-	data: {id: 1, name: "test"}
+  method: "PUT",
+  url: "/api/v1/users/:id",
+  body: {id: 1, name: "test"}
```

Comment:
Aren't the URL parameters are only replaced from `params` and not from `body`?

Reply:
Yep, this needs fixed.

Reply:
OK